crypt.go

```diff
@@ -641,13 +693,13 @@ type InitializeLUKS2ContainerOptions struct {
 	// expressed in multiples of 1024 bytes. The value must be aligned to
 	// 4096 bytes, with the maximum size of 128MB.
 	KeyslotsAreaKiBSize int
+
+	// KDFOptions sets the KDF options for the initial keyslot. If this
+	// is nil then the defaults are used.
+	KDFOptions *KDFOptions
 }
 
 func validateInitializeLUKS2Options(options *InitializeLUKS2ContainerOptions) error {
-	if options == nil {
-		return nil
-	}
-
```

Comment:
in master this has not been dropped, as if nil options are ok. Because they are all initialized at some later point before use.